.github/workflows/dependency-submission.yml

```diff
@@ -28,3 +28,6 @@ jobs:
         # https://github.com/scalacenter/sbt-dependency-submission/releases
         # v2.1.0
         uses: scalacenter/sbt-dependency-submission@573f2f9ac18dd26794bd0c5b26b71e880467b608
+        with:
+          modules-ignore: examples_2.13 examples_2.12
+          configs-ignore: test It
```

Comment:
@ennru should it also ignore https://github.com/akka/akka-projection/tree/main/samples ?

Reply:
It doesn't see it. This action uses sbt dependency tree only.